### PR TITLE
Fix build of Ubuntu Image: Update WebP to v.7, fix Symlink

### PR DESCRIPTION
**Pull request by 0xfacade**
Thank you for putting together this very nice image!

This should allow building the Ubuntu version again.

### CODE REVIEW

File: ubuntu/Dockerfile [hunk 1]
```diff
@@ -70,7 +70,7 @@ RUN set -ex \
         /opt/opencv-${OPENCV_VERSION} \
     && make -j$(nproc) \
     && make install \
-    && ln -s /usr/local/lib/python3.9/site-packages/cv2/python-3.9/cv2.cpython-39-x86_64-linux-gnu.so /usr/local/lib/python3.9/site-packages/cv2/python-3.9/cv2.so \
+    && ln -s /usr/local/lib/python3.10/dist-packages/cv2/python-3.10/cv2.cpython-310-x86_64-linux-gnu.so /usr/local/lib/python3.10/dist-packages/cv2/python-3.10/cv2.so \
```
Comment (0xfacade) [inline ubuntu/Dockerfile:73]: I'm not sure why this is necessary. Perhaps you could explain? Thanks.

Comment (0xfacade) [reply ubuntu/Dockerfile:73]: Also, note that the Python version is now Python 3.10

Comment (gigante) [reply ubuntu/Dockerfile:73]: this symlink helps python to find cv2 lib

**Comment by gigante** [comment]
Thanks, @0xfacade 👍

**Pull request merged**